Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY=test
```

Comment:
My bad

On Tue, May 24, 2016, 10:15 AM Tommy Brunn <notifications@github.com> wrote:

> In Makefile
> <https://github.com/klarna/ui-react-components/pull/27#discussion_r64346450>
> :
>
> > @@ -0,0 +1,10 @@
> > +.PHONY=test
>
> https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/klarna/ui-react-components/pull/27/files/8cc3e28655a5a3e7f2c0d3ecd06e872238cc28ca#r64346450>
>